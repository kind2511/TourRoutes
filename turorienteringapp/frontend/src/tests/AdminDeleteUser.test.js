import React from 'react';
import { render } from '@testing-library/react';
import DeleteUserButton from '../components/AdminDeleteUser';

test('DeleteUserButton renders without crashing', () => {
  render(<DeleteUserButton userId={1} onDelete={() => {}} />);
});

//------------------------------------------------->

// Snapshot test for the DeleteUserButton component
test('DeleteUserButton component snapshot', () => {
  const { asFragment } = render(<DeleteUserButton userId={1} onDelete={() => {}} />);
  expect(asFragment()).toMatchSnapshot();
});