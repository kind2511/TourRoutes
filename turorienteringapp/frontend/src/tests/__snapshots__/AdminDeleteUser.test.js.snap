// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DeleteUserButton component snapshot 1`] = `
<DocumentFragment>
  <button
    class="admin-users-delete-button"
  >
    Delete
  </button>
</DocumentFragment>
`;
